forest/src/cli/config.rs

```diff
@@ -9,6 +9,32 @@ use std::sync::Arc;
 
 use super::client::Client;
 
+/// Structure that defines daemon configuration when process is detached
+#[derive(Deserialize, Serialize, PartialEq)]
+pub struct DaemonConfig {
```

Comment:
can those be `PathBuf`s or are `String`s imposed by the library?

Reply:
Good idea. No the crate works later with the `File` type. `PathBuf` is much cleaner.